axelrod/tests/strategies/test_axelrod_second.py

```diff
@@ -930,3 +930,33 @@ def test_strategy(self):
         actions = [(C, C), (C, D), (C, C), (C, D), (D, C), (C, D)]
         self.versus_test(axelrod.Alternator(), expected_actions=actions, seed=4, attrs={"flack": 5./16.})
 
+
+class TestLeyvraz(TestPlayer):
+    name = 'Leyvraz'
+    player = axelrod.Leyvraz
+    expected_classifier = {
+        'memory_depth': 3,
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
```

Comment:
Could we add a `test_init_of_dict(self)` just to add a second anchor for the internal dictionary.

```
def test_init_of_dict(self):
    player = self.player()
    expected_prob_coop = {...}
    self.assertEqual(player.prob_coop, expected_proob_coop)
```